src/app/learn/reference/extensibility/index.html

```diff
@@ -16,80 +16,96 @@
-
-As an example of whatever file you choose to satisfy the last requirement above:
-
-```
-module.exports = {
-  runCommand(command, argv) {
-    switch (command) {
-      case 'customCommand':
-        // Do something here
-        return true;
-
-      default:
-        return false;
-    }
-  }
-}
-```
-
-## SKY UX Builder Plugins
-
-Add the package as a dependency in `package.json`.
-
-```
-{
-  "dependencies": {
-    "skyux-builder-plugin-example": "1.0.0"
-  }
-}
-```
-
-In the `skyuxconfig.json` file, specify the module in the `plugins` array.  For example:
-
-```
-{
-  "plugins": [
-    "skyux-builder-plugin-example"
-  ]
-}
-```
-
-Your package must contain one of the following:
-  - The `package.json` must specify the `main` property which points to a valid JavaScript file.
-  - An `index.js` file in the root.
-
-As an example of whatever file you choose to satisfy the above requirement:
-
-```
-module.exports = {
-  // content = a string representation of the file's contents
-  // resourcePath = the absolute path to the file
-  // skyPagesConfig = the prepared config that includes skyuxconfig.json and builder's defaults
-  preload(content, resourcePath, skyPagesConfig) {
-    return content;
-  }
-}
-```
+SKY UX Builder is intended to be listed as a `devDependency` in the `package.json` file for every SPA. Builder handles the majority of `skyux` commands, including testing, serving, and building, and it wraps up the logic and configuration for other tools such as the Angular Compiler, Webpack, Karma, and Protractor.
+
+## SKY UX CLI plugins
+
+SKY UX CLI is extensible. Follow these guidelines to create custom commands and modules to handle those commands:
+
+<ul>
+  <li>
+    Install the module globally or locally.
+  </li>
+  <li>
+    Start the module name with `@skyux-sdk/builder`.
+  </li>
+  <li>
+    Enable the module for use with `require()` by either specifying a valid JavaScript file with the `main` property in the module's `package.json` file or by including an `index.js` file in the module's root.
+  </li>
+  <li>
+    Expose a `runCommand` method in the module. This method is called with two parameters: the requested command and an arguments object. It returns a boolean value to indicate whether it successfully handled the command.
+    <br>
+    <br>
+    <sky-code-block>
+      module.exports = {
+        runCommand(command, argv) {
+          switch (command) {
+            case 'customCommand':
+              // Do something here
+              return true;
+      
+            default:
+              return false;
+          }
+        }
+      }
+    </sky-code-block>
+  </li>
+</ul>
+
+## SKY UX Builder plugins
+
+SKY UX Builder is extensible. Follow these guidelines to create custom packages:
```

Comment:
I like the addition of these sentence, but maybe we could add how it's extensible.  "SKY UX Builder is extensible but in a different way.  Each file that's parsed as part of your project can be modified at build time.  This technique can be very useful for dynamically injecting common boilerplate behavior."